lazysignup/views.py

```diff
@@ -9,14 +9,22 @@
 from django.contrib.auth import authenticate
 from django.contrib.auth import login
 
+try:  # pragma: no cover
+    from django.utils.module_loading import import_string
+except ImportError:  # pragma: no cover
+    from django.utils.module_loading import import_by_path as import_string
+
+
 from lazysignup.decorators import allow_lazy_user
 from lazysignup.exceptions import NotLazyError
-from lazysignup.forms import UserCreationForm
 from lazysignup.models import LazyUser
+from lazysignup import constants
+
+CREATION_FORM = import_string(constants.LAZYSIGNUP_CUSTOM_USER_CREATION_FORM)
 
 
 @allow_lazy_user
-def convert(request, form_class=UserCreationForm,
+def convert(request, form_class=CREATION_FORM,
```

Comment:
I'm not keen on this change - consider that this form might be generated. It also means we have to support another setting in settings.py (something I'm keen to avoid).